var/spack/repos/builtin/packages/slurm/package.py

```diff
@@ -213,6 +216,9 @@ def configure_args(self):
         else:
             args.append("--without-pmix")
 
+        if "+nvml" in spec:
+            args.append("--with-nvml={0}".format(spec["cuda"].prefix))
```

Comment:
We are also in the process of switching to use of f-strings where appropriate (https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals).

```suggestion
            args.append(f"--with-nvml={spec["cuda"].prefix}")
```

Reply:
I think it requires two different types of quotes. There are a lot more instances of this idiom and the "in spec" idiom in this file.  Convert them all?  I don't think it would be an improvement to mix idioms in the same file.

Reply:
Oops.  You are correct that two different quotes are needed!